### PR TITLE
create init for tellor flex

### PR DESCRIPTION
**Pull request by tallywiesenberg**
points tellorflex to governance

can only be called once

### CODE REVIEW

File: contracts/TellorFlex.sol [hunk 1]
```diff
@@ -71,17 +73,32 @@ contract TellorFlex {
...
+    function init(address _governanceAddress) external {
+        require(msg.sender == owner);
+        require(initted == 0);
+        require(
+            _governanceAddress != address(0),
+            "governance address can't be zero address"
+        );
+
+        governance = _governanceAddress;
+        initted = 1;
```
Comment (tkernell) [inline contracts/TellorFlex.sol:98]: couldn't we remove `initted` and just `require(governance == address(0))` so that `init` can only be called once?

Comment (tkernell) [reply contracts/TellorFlex.sol:98]: and then we'd remove the `_governance` argument from the constructor

Comment (tallywiesenberg) [reply contracts/TellorFlex.sol:98]: good call, will do!

**Review by themandalore — APPROVED** [review]

**Comment by themandalore** [comment]
@tallywiesenberg , a whole bunch of conflicts...

**Comment by tallywiesenberg** [comment]
- added some mock funcitons in `MockGovernance.sol` to mock governance calls to change  the `reporterLock`, `stakeAmount`, and `governanceAddress.
- updated tests to call the `init()` function in the `beforeEach`
- implemented @tkernell 's recommendation to remove the `initted` var

File: contracts/TellorFlex.sol [hunk 2]
```diff
@@ -81,24 +82,72 @@ contract TellorFlex {
...
+        );
+
+        governance = _governanceAddress;
+    }
+
+    /**
+     * @dev Changes governance address
+     * @param _newGovernanceAddress new governance address
+     */
+    function changeGovernanceAddress(address _newGovernanceAddress) external {
```
[the remark this answers is not in the document]
Comment (tallywiesenberg) [reply contracts/TellorFlex.sol:116]: aha I gotta remove these functions and their tests then

**Comment by tallywiesenberg** [comment]
Removed the deprecated functions + their tests

**Pull request merged**